experiments/ClimaEarth/setup_run.jl

```diff
@@ -673,12 +671,8 @@ function step!(cs::CoupledSimulation)
     ## update the surface fractions for surface models
     FieldExchanger.update_surface_fractions!(cs)
 
-    ## update the coupler with the new surface properties
-    FieldExchanger.import_combined_surface_fields!(cs.fields, cs.model_sims)
-    ## update the coupler with the new atmospheric properties
-    FieldExchanger.import_atmos_fields!(cs.fields, cs.model_sims)
-    ## update the model simulations with the new coupler fields
-    FieldExchanger.update_model_sims!(cs.model_sims, cs.fields)
```

Comment:
Let's also maybe remove these functions from the exported list and public APIs (if they are never called individually)

Reply:
In the initial exchange in `setup_run.jl`, we still call `import_atmos_fields!` + `update_model_sims!` individually right after the radiation callback, to update the coupler fields and surface models with variables set in the atmos callback. Since the surface models haven't changed, there's no need to call `import_combined_surface_fields!` (unless maybe we want to recompute surface humidity)

Reply:
If we can substitute that with `exchange!`, I'd do it for the sake of simplicy, even if it is less efficient (and performance doesn't as much at init)

Reply:
okay! I'll use `exchange!` there too and remove the other functions from API docs